jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/messageframe/MessageFrameCoder.java

```diff
@@ -0,0 +1,122 @@
+package us.dot.its.jpo.ode.messageframe;
+
```

Comment:
This class has a lot in common with BsmCoder class. Given that MessageFrame basically wraps a BSM, I think MessageFrame can actually extend BsmCoder. Let's refactor it in the next sprint.